src/views/DigitalCredentials.vue

```diff
@@ -139,6 +149,49 @@ export default class DigitalCredentials extends Vue {
   cancel (): void {
     this.$router.push({ path: `/${Routes.DIGITAL_CREDENTIALS}` })
   }
+
+  async getCredentials (): Promise<void> {
```

Comment:
Functions called from the template should be protected. Please check all in this file.

Reply:
Updated!